pkg/apis/crd/v1alpha1/types.go

```diff
@@ -289,6 +292,14 @@ type Advertisements struct {
 	Egress *EgressAdvertisement `json:"egress,omitempty"`
 }
 
+type Confederation struct {
+	// Identifier specifies the confederation's ASN.
+	Identifier int32 `json:"identifier,omitempty"`
+
+	// MemberASNs lists the ASNs of other members in the confederation.
+	MemberASNs []int32 `json:"peers,omitempty"`
```

Comment:
just noticed that the JSON field name is wrong. It should be `memberASNs`.

Reply:
My bad. I have corrected it.